src/sentry/scim/endpoints/members.py

```diff
@@ -81,7 +86,54 @@ def _should_delete_member(self, operation):
                 return True
         return False
 
+    @extend_schema(
+        operation_id="Query an Individual Organization Member",
+        parameters=[GLOBAL_PARAMS.ORG_SLUG, SCIM_PARAMS.MEMBER_ID],
+        request=None,
+        responses={
+            200: inline_serializer(
```

Comment:
if inline serializers don't work for whatever reason, manual JSON can always be written, so parity with the current implementation is achieved.

Reply:
note: auto-doc'ing our serializers is out of scope in this PR, you can see in the proposal some possible paths for that.

Reply:
would this have to defined on every method? this _feels_ awfully expensive hard to maintain if everythings going to be this much boilerplate

Reply:
@dcramer I have been doing a spike on the serializer front and have a solution that simply requires you to type the response serializer `serialize` method -- will be putting up a PR for an example of that shortly.

Reply:
See https://github.com/getsentry/sentry/pull/31067